Modules/Registration/Common/wrapping/itkPointSetToPointSetMetric.wrap

```diff
@@ -12,4 +12,4 @@ itk_wrap_class("itk::PointSetToPointSetMetric" POINTER_WITH_2_SUPERCLASSES)
       itk_wrap_template("PS${ITKM_${t}}23" "itk::PointSet< ${ITKT_${t}},2 >, itk::PointSet< ${ITKT_${t}},3 >")
     endforeach()
   endif()
-itk_end_wrap_class()
\ No newline at end of file
+itk_end_wrap_class()
```

Comment:
These commits should be squashed, upon merge if not before.

Reply:
Agree!